rocAL/rocAL/include/api/rocal_api_augmentation.h

```diff
@@ -146,14 +146,18 @@ extern "C" RocalTensor ROCAL_API_CALL rocalBrightness(RocalContext context, Roca
                                                       RocalTensorOutputType rocal_tensor_output_type = ROCAL_UINT8);
 
 /// Accepts U8 and RGB24 inputs
-/// \param context
-/// \param input
-/// \param shift
-/// \param is_output
+/// \param context Rocal context
+/// \param input Input Rocal tensor
+/// \param is_output is the output tensor part of the graph output
+/// \param alpha controls contrast of the image
+/// \param beta controls brightness of the image
+/// \param tensor_output_layout the layout of the output tensor
+/// \param tensor_output_datatype the data type of the output tensor
 /// \return
-extern "C"  RocalTensor  ROCAL_API_CALL rocalBrightnessFixed(RocalContext context, RocalTensor input,
-                                                            float alpha, float beta,
-                                                            bool is_output );
+extern "C" RocalTensor ROCAL_API_CALL rocalBrightnessFixed(RocalContext context, RocalTensor input,
+                                                           float alpha, float beta, bool is_output,
```

Comment:
I thnik we need to modify it to support different alpha and beta for each image in tensor, in other words alpha and beta has to be an array instead

Reply:
If we pass alpha and beta as an array it will be the same for every iteration even though its different for every image in a batch.
If we need to have different value for each image then we need to pass it as rocALTensor and update the values in python using external_source.

Reply:
Ok. For this version we will merge this since it is the same as image based functionality